hack/authentication.yaml

```diff
@@ -0,0 +1,2 @@
+  authentication:
+    aws: {}
```

Comment:
[these](https://pkg.go.dev/k8s.io/kops/pkg/apis/kops?tab=doc#AwsAuthenticationSpec) are the fields you can add here.

```yaml
  authentication:
    aws:
      image: foo
```